### ISSUE TITLE
ISO WL Bounds Mismatch

### ISSUE THREAD

**Issue by jdtsmith** [PAHFIT/pahfit#239]
@jancami / @els1, it looks like the ISO instrument pack doesn't quite comport with the sample Orion spectrum. 

```python
Input wavelength bounds exceed instrument segment(s) by more than 1%:
	Segment(s) cover: 2.38-45.20, wave_bounds: 2.37-45.35
```

I.e. the wavelength range in our Orion spectrum is >~1% larger than the instrument pack specifies.  Could you take a look and see if there's an obvious solution (padding the final segment?).

**Comment by BethanyRS** [comment]
Just commenting as a reminder because I ran into this issue again. Would it be a good idea to temporarily remove the data files that cause this issues so that new users experimenting with PAHFIT don't have to worry about which files they select in the data folder?

**Comment by jdtsmith** [comment]
Better would be to fix it in the instrument package iso.yaml file.  Can you bug @jancami?

**Comment by jdtsmith** [comment]
I think this is fixed by #261.
